ee/clickhouse/queries/trends/clickhouse_trends.py

```diff
@@ -63,15 +103,19 @@ def run(self, filter: Filter, team: Team, *args, **kwargs) -> List[Dict[str, Any
         if filter.formula:
             return handle_compare(filter, self._run_formula_query, team)
 
-        result = []
         for entity in filter.entities:
             if entity.type == TREND_FILTER_TYPE_ACTIONS:
                 try:
                     entity.name = actions.get(id=entity.id).name
                 except Action.DoesNotExist:
-                    continue
-            entities_list = handle_compare(filter, self._run_query, team, entity=entity)
-            result.extend(entities_list)
+                    return []
+
+        if len(filter.entities) == 1 or filter.compare:
```

Comment:
For formula, clickhouse does the calculation for each day. I guess we could do that in python but that would be quite a lot of effort.

For compare, because of the constraints of not doing sql inside a thread the logic would have been kind of complicated, and it's not a feature that's used a ton + I wanted to get it out quick.